From 5c91f4ddbed439de473d7839d3ba86b4d0a139f9 Mon Sep 17 00:00:00 2001
From: Firelight Flagboy <firelight.flagboy@gmail.com>
Date: Sun, 18 Feb 2024 14:14:23 +0100
Subject: [PATCH] Use local history file in docs

---
 docs/history.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/history.rst b/docs/history.rst
index f5285bc0f..9606e7c26 100644
--- a/docs/history.rst
+++ b/docs/history.rst
@@ -2,4 +2,4 @@
 
 .. _doc_history:
 
-.. include:: ../HISTORY.rst
+.. include:: ./HISTORY.rst
-- 
2.42.0

